Sources/Build/BuildPlan/BuildPlan.swift

```diff
@@ -526,7 +526,7 @@ public class BuildPlan: SPMBuildCore.BuildPlan {
         for target in targets {
             switch target {
             case .swift(let targetDescription):
-                arguments += ["-I", targetDescription.moduleOutputPath.parentDirectory.pathString]
+                arguments += ["-I", targetDescription.modulesPath.pathString]
```

Comment:
These paths aren't the same I assume? If so, shouldn't this change have some impact on the test suite, either requiring to update tests or add a new one?

Reply:
Right, I'm worried about the fact that the remaining minority of cases doesn't seem to be covered with tests, otherwise this change would've broken such tests?